model/task/task.go

```diff
@@ -1049,7 +1049,17 @@ func AbortBuild(buildId string) error {
 		},
 		bson.M{"$set": bson.M{AbortedKey: true}},
 	)
-	return errors.WithStack(err)
+	if err != nil {
+		return errors.Wrap(err, "error setting aborted statuses")
+	}
+	ids, err := FindAllTaskIDsFromBuild(buildId)
+	if err != nil {
+		return errors.Wrap(err, "error finding tasks by build id")
+	}
+	if len(ids) > 0 {
+		event.LogManyTaskAbortRequests(ids, evergreen.GithubPatchUser)
```

Comment:
Why is this GithubPatchUser? "mci" is probably more appropriate if we don't have a good option

Reply:
done